.vsts-ci/templates/pushPhase.yml

```diff
@@ -1,30 +1,73 @@
 parameters:
-  vmImage: 'ubuntu-18.04'
-  parentJobs: []
-  matrix:
+  - name: vmImage
+    default: 'ubuntu-18.04'
+  - name: channel
+    type: string
+  - name: release
+    default: 'private'
```

Comment:
yeah, this will be overwritten, just didn't want someone calling this in a new scenario and not providing the parameter and releasing to stable.

Think of this as an internal function.   This function defaults to private... but the public one for official release defaults to stable, the public function calls this internal function.